geopy/geocoders/googleplaces.py

```diff
@@ -50,16 +57,54 @@ def __init__(
             through the specified proxy. E.g., {"https": "192.0.2.0"}. For
             more information, see documentation on
             :class:`urllib2.ProxyHandler`.
+
+            .. versionadded:: 1.12.1
+
+        :param string client_id: If using premier, the account client id.
+
+        :param string secret_key: If using premier, the account secret key.
+
+        :param string channel: If using premier, the channel identifier.
         """
         super(GooglePlaces, self).__init__(
             scheme=scheme, timeout=timeout, proxies=proxies, user_agent=user_agent
         )
+        if client_id and not secret_key:
+            raise ConfigurationError('Must provide secret_key with client_id.')
+        if secret_key and not client_id:
```

Comment:
But isn't the point that this error is raised only if one is defined without the other?
If both are None, the code continues and uses api_key instead.